src/main/resources/admin/tools/content-type-manager/components/component-file-handler.html

```diff
@@ -140,34 +143,82 @@
         ///
 
         setBorderOfVcardsWithErrors() {
-          setTimeout(() => {
+          // Get all error DOM elements
+          const getAllDOMelementsWithErrors = () =>
             Array.from(
               document.getElementsByClassName("v-messages error--text")
-            ).forEach((div) => {
-              let i = 0,
-                el = div;
+            );
+
+          // For each error elements, find it's Vcard father and colorize it
+          const colorizeElementsWithErrors = (elements) => {
+            elements.forEach((div) => {
+              let i = 0;
+              let el = div;
               const m = 10;
 
               while (el && i < m) {
-                if (
-                  Array.from(el.classList).indexOf("v-card") !== -1 &&
-                  Array.from(el.classList).indexOf("v-sheet") !== -1
-                ) {
+                if (isElementVCard(el)) {
                   break;
                 } else {
                   el = el.parentElement;
                   i += 1;
                 }
               }
 
-              el.style.border = `2px solid ${this.errorVcardColor}`;
+              el.style.border = `2px solid ${this.error.borderColor}`;
+              el.style.backgroundColor = this.error.backgroundColor;
             });
+
+            return elements;
+          };
+
+          // Expand all parent elements of el
+          const expandAllParentElements = (elements) => {
+            elements.forEach((el) => {
+              getAllElementParents(el)
+                .filter((el) => isElementVCard(el))
+                .forEach((el) => {
+                  el.__vue__.$parent.showConfig = true;
+                });
+            });
+          };
+
+          ///
+
+          // Check if an element is a vuetify card
+          const isElementVCard = (el) => {
+            return (
+              (((el.__vue__ || {}).$options || {})._componentTag || "") ===
+              "v-card"
+            );
+          };
+
+          // Get all father parents of el
+          const getAllElementParents = (el) => {
+            return [
+              ...(function* (e) {
```

Comment:
I find this a bit confusing. Why setup an iterator that always and immediatly returns all elements? 
This is a cool iterator for getting the next parent element in context of the current iterator element.